.github/workflows/pylint.yml

```diff
@@ -6,9 +6,7 @@ jobs:
   Black:
     runs-on: ubuntu-latest
     steps:
-      - uses: snow-actions/sparse-checkout@v1.1.0
-        with:
-          patterns: "mqt"
+      - uses: actions/checkout@v3
       - uses: actions/setup-python@v2
```

Comment:
not quite sure why this hasn't been caught by dependabot (maybe check that configuration and compare it to the configuration in our other projects), but there is a more recent version of that action. IIRC it's even `v4` now. Please also check in the other GitHub Actions.

Reply:
Apparently it is still v3 (https://github.com/actions/checkout)

Reply:
I was referring to setup-python ;o)

Reply:
done